pythonAPIClient/client.py

```diff
@@ -297,13 +298,154 @@ def usage_by_client(self, clientID, owner):
         return response.json()['api_usage']
 
     # TODO: add a camera to database
-    def add_camera(self, camera):
+    def add_camera(self):
         pass
 
-    # TODO: update a camera in database
-    # replace others with desired field names
-    def update_camera(self, camID, others):
-        pass
+    def update_camera(self, cameraID, camera_type=None, is_active_image=None,
```

Comment:
can we please have kargs integrating here.

Reply:
Yeah Jayden will implement that as soon as this gets merged